src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesService.kt

```diff
@@ -209,14 +209,8 @@ class ActivitiesService(
     )
   }
 
-  private fun toNomisActivityDescription(activityDescription: String, scheduleDescription: String): String {
-    var description = "SAA $activityDescription"
-      .let { it.substring(0, min(it.length, 40)).trim() }
-    if (description.length < 39) {
-      description += " ${scheduleDescription.let { it.substring(0, min(it.length, 39 - description.length)) }}"
-    }
-    return description
-  }
+  private fun toNomisActivityDescription(activityDescription: String): String =
+    "SAA " + activityDescription.substring(0, min(36, activityDescription.length))
```

Comment:
The activity model has changed to 1 schedule per activity whose descriptions are the same